app/onepassword_events_api/default/data/ui/views/item_usage_dashboard.xml

```diff
@@ -14,8 +14,8 @@
       <fieldForValue>index</fieldForValue>
       <search>
         <query>index=* sourcetype="1password:insights:item_usages" | dedup index | table index</query>
-        <earliest>-24h@h</earliest>
-        <latest>now</latest>
+        <earliest>0</earliest>
```

Comment:
I've changed the time-frame for this search, because it will yield no index name for the `sourcetype` if there are no reported events in the last `24hs`, making impossible to use the search on that scenario.